mypyc/test-data/run-sets.test

```diff
@@ -116,6 +116,71 @@ s = {1, 2, 3}
 update(s, [5, 4, 3])
 assert s == {1, 2, 3, 4, 5}
 
+[case testFrozenSets]
+from typing import FrozenSet, List, Any, cast
+def instantiateLiteral() -> FrozenSet[int]:
+    return frozenset((1, 2, 3, 5, 8))
+
+def emptyFrozenSet1() -> FrozenSet[int]:
+    return frozenset()
+
+def emptyFrozenSet2() -> FrozenSet[int]:
+    return frozenset(())
+
+def fromIterator() -> List[FrozenSet[int]]:
+    a = frozenset([1, 3, 5])
+    b = frozenset((1, 3, 5))
+    c = frozenset({1, 3, 5})
+    d = frozenset({1: '1', 3: '3', 5: '5'})
+    e = frozenset(x for x in range(1, 6, 2))
+    f = frozenset((x for x in range(1, 6, 2)))
+    return [a, b, c, d, e, f]
+
+def fromIterator2() -> FrozenSet[int]:
+    tmp_list = [1, 2, 3, 4, 5]
+    return frozenset((x + 1) for x in ((y * 10) for y in (z for z in tmp_list if z < 4)))
+
+def castFrozenSet() -> FrozenSet[int]:
+    x: Any = frozenset((1, 2, 3, 5, 8))
+    return cast(FrozenSet, x)
+
+[file driver.py]
```

Comment:
Style: The current preference is omit `driver.py` in run tests, and add `test_*` functions with assertions in the compiled code. Not a big deal though.